httpx/dispatch/proxy_http.py

```diff
@@ -84,12 +84,14 @@ def __init__(
             credentials, _, authority = url.authority.rpartition("@")
             self.proxy_url = url.copy_with(authority=authority)
 
-    async def acquire_connection(self, origin: Origin) -> HTTPConnection:
+    async def acquire_connection(
+        self, origin: Origin, timeout: TimeoutTypes = UNSET
+    ) -> HTTPConnection:
         if self.should_forward_origin(origin):
             logger.debug(
                 f"forward_connection proxy_url={self.proxy_url!r} origin={origin!r}"
             )
-            return await super().acquire_connection(self.proxy_url.origin)
+            return await super().acquire_connection(self.proxy_url.origin, timeout)
```

Comment:
Should we pass `timeout` as a keyword argument for consistency?

```suggestion
            return await super().acquire_connection(self.proxy_url.origin, timeout=timeout)
```

Reply:
In `connection_pool.py` we pass both origin and timeout as keyword arguments, maybe we should change both then?

Reply:
Yep, sounds good.

Reply:
Changed